sdk/test-utils/recorder/package.json

```diff
@@ -5,7 +5,6 @@
   "main": "dist/index.js",
   "module": "dist-esm/src/index.js",
   "browser": {
-    "dist/index.js": "./dist-browser/azure-test-utils-recorder.js"
```

Comment:
@HarshaNalluru I remember you have plan to do browser mapping for the nock/nise recoders?

Reply:
Just tested on my local.. removing this line still works fine somehow.

Reply:
I have an ongoing PR to separate node and browser codes, but this PR is fine.